Packages/Tracking/Core/Runtime/Ultraleap.Tracking.Core.asmdef

```diff
@@ -4,7 +4,8 @@
     "references": [
         "GUID:2ee0542102ac1084e8d9a910885c1e81",
         "GUID:ba171b3dd2a51234ab864770f99741a5",
-        "GUID:e40ba710768534012815d3193fa296cb"
+        "GUID:e40ba710768534012815d3193fa296cb",
+        "GUID:ce522b6ed64c8be4c989a1d26d0e3275"
```

Comment:
I think we should change this asmdef to be specific to the subsystem work and probably make it similar to the OpenXR one in that it will only exist if XRHands is in your project